contracts/modules/assessment/Assessment.sol

```diff
@@ -510,6 +489,26 @@ contract Assessment is IAssessment, MasterAwareV2 {
   function changeDependentContractAddress() external override {
```

Comment:
would be good to run the fork to see everything works well in tandem here the fields are populated correctly after upgrade

Reply:
initialize() wasn't even called on the existing fork test FYI